index.html

```diff
@@ -3447,6 +3447,19 @@ <h3>Syntactic Sugar</h3>
         </section>
 
       </section>
+
+      <section>
+        <h3>Representations of Verifiable Credentials</h3>
+
+        <p>
+Verifiable Credentials can utilize representations other than the standard one defined in this specification (<code>application/vc+ld+json</code>).
+They can do so provided that a mapping is defined between that representation and the standard one defined in this specifiction,
+so that deployments that choose to do so can map that representation to the standard one and process it accordingly.
+This mapping definition should state whether it is unidirectional or bidirectional.
+A media type should be defined for each Verifiable Credential representation so that protocols can use the media type to distinguish between representations.
```

Comment:
```suggestion
A media type should be defined for each Verifiable Credential representation so that it can be used to distinguish between representations.
```

I don't think it is only the protocols that will use media types to distinguish? (I am not too strongly opinionated on this)

Reply:
```suggestion
A media type should be defined for each Verifiable Credential representation, such that these media types can be used to distinguish between representations.
```